src/docs/markdown/caddyfile/options.md

```diff
@@ -969,6 +971,15 @@ Enables Prometheus metrics collection; necessary before scraping metrics. Note t
 }
 ```
 
+You can add or remove the per_host option to separate metrics for each host rather than for the entire caddy server.
```

Comment:
This is not accurate. When the `per_host` option is added, it adds a label of the hostname for which the metrics is recorded. This way you can see metrics for a particular hostname.

```suggestion
You can add the `per_host` option to label metrics with the host name of the metric.
```